fingerprint/search_by_fingerprint.md

```diff
@@ -0,0 +1,15 @@
+# Search By Fingerprint
```

Comment:
@slubwama move this to the same document as add_patient_fingerprint there is no need to split them